content/en/docs/refguide/modeling/application-logic/workflows/workflow-elements/boundary-events/timer-event.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Timer Event"
+url: /refguide/timer-event/
+weight: 10
+---
+
+{{% alert color="info" %}}
+This feature is currently in public beta as of version 10.15. Please note that certain features or attributes may be subject to change and may contain bugs.{{% /alert %}}
+
+## Introduction
+
+The **Timer Event** enables the timed-execution of alternative flows when the guarded activity is exceeding the set timer.
+
+## Properties
+
+**Timer Event** properties consist of the following sections:
+
+* [General](#general)
+* [Timer](#timer)
+* [Common](#common)
+
+### General Section {#general}
+
+The **Caption** describes what happens in this element. It is displayed on the workflow element to make the **Timer** event easier to read and understand.
```

Comment:
If we don't treat it as a workflow "element" we probably should mention it being such. So, we should reword it into something like "describes what happens in this boundary event"